### DISCUSSION TITLE
Toe brakes button assignment

### DISCUSSION THREAD

**Discussion by ZaccJacc** [magknight/787aviatorsTracker#581] [Ideas]
For people who maybe can't afford rudder pedals or cannot use them for some reason, the only alternative is button assignment. Obviously, the setting used is 'Hold brakes at normal/max effort'. In many addon aircraft, this acts on the toe brakes as normal pedals would, however in the Magknight this clicks the parking brake halfway then back off again. This means you end up with an annoying little click sound in the background whenever you try to brake. Could this hold brakes setting be re-assigned so it acts upon the toe brakes instead? 

Thanks :)

**Comment by aeroniemi** [comment]
We kinda did this - both commands now operate the parking brake
